test/lib/tokenFactory.test.js

```diff
@@ -31,11 +40,21 @@ describe('Token Factory', () => {
             useCleanCache: true,
             warnOnUnregistered: false
         });
+        generateTokenMock = {
+            generateValue: sinon.stub(),
+            hashValue: sinon.stub()
```

Comment:
The mock that's here is fine. `generateToken.test.js` will handle how that particular test works out.